pkg/advisory/index_adapter.go

```diff
@@ -0,0 +1,58 @@
+package advisory
```

Comment:
Looks like we don't use boilerplate in these repo, and when we do it's interesting. No action necessary, just an observation.

Reply:
👍